code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -1,22 +1,31 @@
-/mob/living/carbon/human/examine(mob/user)
+// Clothing flags, organized in roughly top-bottom
+#define EXAMINE_SKIPHELMET			0x0001
+#define EXAMINE_SKIPEARS			0x0002
+#define EXAMINE_SKIPEYEWEAR			0x0004
+#define EXAMINE_SKIPMASK			0x0008
+#define EXAMINE_SKIPJUMPSUIT		0x0010
+#define EXAMINE_SKIPTIE				0x0020

```

Comment:
Maybe these should be moved to a defines/examine.dm. They might be useful elsewhere or on other mob types.